src/shared/components/FormField/FormField.tsx

```diff
@@ -1,17 +1,33 @@
 import React from 'react'
 
-import { ChildrenWrapper, FormFieldDescription, FormFieldTitle, FormFieldWrapper } from './FormField.style'
+import {
+  ChildrenWrapper,
+  FormFieldDescription,
+  FormFieldHeader,
+  FormFieldTitle,
+  FormFieldWrapper,
+} from './FormField.style'
+
+import { Text } from '../Text'
 
 export type FormFieldProps = {
   title: string
+  optional?: boolean
   description?: string | string[]
   className?: string
 }
 
-export const FormField: React.FC<FormFieldProps> = ({ children, title, description, className }) => {
+export const FormField: React.FC<FormFieldProps> = ({ children, title, description, className, optional }) => {
   return (
     <FormFieldWrapper className={className}>
-      <FormFieldTitle variant="h6">{title}</FormFieldTitle>
+      <FormFieldHeader>
+        <FormFieldTitle variant="h6">{title}</FormFieldTitle>
+        {optional && (
+          <Text variant="body2" secondary>
+            (Optional)
+          </Text>
+        )}
+      </FormFieldHeader>
```

Comment:
This doesn't produce a great visual effect - the two text nodes aren't properly centered. It's not your fault - just issues with the font in the app and Figma. Will discuss with designers what we should do about this

<img width="1153" alt="Screenshot 2021-09-03 at 11 04 30" src="https://user-images.githubusercontent.com/12646744/131980598-31dc2c85-46e0-43d7-8d01-d4f8d5020c88.png">
<img width="547" alt="Screenshot 2021-09-03 at 11 04 42" src="https://user-images.githubusercontent.com/12646744/131980606-f0390be1-4782-4548-9952-674351699a34.png">

Reply:
As a workaround, I used `align-items: baseline` instead of `center`. I think it looks slightly better now.